v1/transform.go

```diff
@@ -235,44 +294,34 @@ func toV2DeviceResources(profile v1Models.DeviceProfile) []models.DeviceResource
 	return resources
 }
 
-func isResourceHidden(resourceName string, profile v1Models.DeviceProfile) bool {
-	// Check whether the resource exists in the CoreCommands, if exists, the resource is not hidden.
-	for _, coreCommand := range profile.CoreCommands {
-		for _, res := range coreCommand.Get.Responses {
-			isContains := contains(res.ExpectedValues, resourceName)
-			if isContains {
-				return false
-			}
-		}
-		isContains := contains(coreCommand.Put.ParameterNames, resourceName)
-		if isContains {
-			return false
+func isV2ResourceHidden(resourceName string, v2deviceCommands []models.DeviceCommand, v1CoreCommands []v1Models.Command) bool {
+	// Skip the checking if the resource exists in the v2 DeviceCommands
+	for _, v2deviceCommand := range v2deviceCommands {
```

Comment:
don't check DeviceCommand, because it is irrelevant.

Reply:
Removed.